### PR TITLE
Hidden folder when trying to make changes

### PR DESCRIPTION
**Pull request by miguelgargallo**
There is a hidden folder that do not allows anyone to change files inside Hidden folder when trying to make changes Relates with #43

### CODE REVIEW

**Comment by m4tt72** [comment]
Hello @miguelgargallo, thanks for the contribution.

I'm not sure what you're trying to achieve in this PR? Could you please provide more information?

**Comment by miguelgargallo** [comment]
Yes, in gitignore there is included /bin, when you try to git clone and deploy and change anything inside the same folder as "bin", it shows you an error not finding "bin" on deploy

**Comment by m4tt72** [comment]
@miguelgargallo please provide more info

**Comment by m4tt72** [comment]
@miguelgargallo the description of this PR is confusing, but I think I stumbled upon the same issue that you faced. The reason why there's a `bin` in `.gitignore` is because I have some scripts inside the `bin` folder that don't necessarily have to make it to the repo, as a consequence to that, the `src/utils/bin` also gets ignored. So as a solution, I suggest you don't remove the `bin` from `.gitignore` but instead either:

1. Change the `bin/` to `/bin/` so git only ignores any folder called `bin` in root dir.
2. Exclude `src/utils/bin` in `.gitignore`

Thanks for your submission.

File: .gitignore [hunk 1]
```diff
@@ -41,7 +41,6 @@ jspm_packages
 
 .env
 
-bin/
```
Comment (m4tt72) [inline .gitignore:44]: instead of removing this line, you need to change it to `/bin/`

**Review by m4tt72 — CHANGES_REQUESTED** [review]

**Comment by m4tt72** [comment]
Closing the PR because [a fix](https://github.com/m4tt72/terminal/pull/45) was necessary and this PR went stale

**Pull request closed without merging**